config/identity/handler/provider-factory/identity.json

```diff
@@ -43,6 +44,7 @@
           "AccessToken": 3600,
           "AuthorizationCode": 600,
           "BackchannelAuthenticationRequest": 600,
+          "ClientCredentials": 600,
```

Comment:
Is that correct? Aren't they longer lived?

Reply:
This is how long the access token lasts that you request with your credentials. The actual credential tokens that we generate are permanent. I just took the default value here of the oidc provider library.